### ISSUE TITLE
[BUG] cudf.Series.to_gpu_array fails when supplying fillna param

### ISSUE THREAD

**Issue by randerzander** [rapidsai/cudf#1048]
From tip of cudf branch-0.6:

1. the [API doc for Series.to_gpu_array](https://rapidsai.github.io/projects/cudf/en/latest/api.html#cudf.dataframe.Series.to_gpu_array) indicates a `str` value should be supplied for fillna.

I expected this to be a numeric type, the value which nulls would be replaced with in the resulting device array.

2. When passing a str _or_ numeric values, I get an error.

Repro:
```
import cudf                                    
s = cudf.Series([0, 1, None, 3])

# same error for fillna=-1
s.to_gpu_array(fillna='-1')
```
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/conda/envs/cudf/lib/python3.7/site-packages/cudf-0.6.0.dev0+724.g454c8df.dirty-py3.7-linux-x86_64.egg/cudf/dataframe/series.py", line 594, in to_gpu_array
    return self._column.to_gpu_array(fillna=fillna)
  File "/conda/envs/cudf/lib/python3.7/site-packages/cudf-0.6.0.dev0+724.g454c8df.dirty-py3.7-linux-x86_64.egg/cudf/dataframe/column.py", line 262, in to_gpu_array
    return self.to_dense_buffer(fillna=fillna).to_gpu_array()
  File "/conda/envs/cudf/lib/python3.7/site-packages/cudf-0.6.0.dev0+724.g454c8df.dirty-py3.7-linux-x86_64.egg/cudf/dataframe/column.py", line 461, in to_dense_buffer
    raise ValueError('invalid for fillna')
ValueError: invalid for fillna
```

**Comment by kkraus14** [comment]
@randerzander does it work if you pass just `s.to_gpu_array(fillana=-1)`?

**Comment by randerzander** [comment]
no, it doesn't. Edited the repro to make it clearer that -1 also doesn't work

**Comment by AK-ayush** [comment]
@randerzander `fillna` in `s.to_gpu_array()` accept only `None` or `'pandas'`. here is doc for the same: https://rapidsai.github.io/projects/cudf/en/latest/api.html#cudf.dataframe.Series.to_gpu_array

And in the code also, fillna is checked against only these two values `{None, 'pandas'}` https://github.com/rapidsai/cudf/blob/branch-0.7/python/cudf/dataframe/column.py#L492

**Comment by AK-ayush** [comment]
I guess, it would be good if this `fillna` accepts scalar value too.
It won't require much effort as I can see.

**Comment by beckernick** [comment]
This was resolved by #1291. Closing.
